codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/sharedstream/person.java

```diff
@@ -25,7 +26,8 @@
 /**
  */
 @Generated("software.amazon.awssdk:codegen")
-public class Person implements SdkPojo, Serializable, ToCopyableBuilder<Person.Builder, Person>, EventStream {
+public class Person implements SdkPojo, Serializable, ToCopyableBuilder<Person.Builder, Person>, StreamBirthsInputEventStream,
+                               EventStream, StreamDeathsInputEventStream {
```

Comment:
Hmm I thought the base POJO class wasn't going to implement all the stream interface anymore?